infra/staging/main.tf

```diff
@@ -119,3 +124,39 @@ resource "google_sql_user" "user" {
   instance = google_sql_database_instance.strapi_db_instance.name
   project = data.google_project.project.project_id
 }
+
+##############
+## Frontend ##
+##############
+
+# Create a new storage bucket for frontend assets
+resource "google_storage_bucket" "call_home_frontend" {
+  project = data.google_project.project.project_id
+  name = var.frontend_bucket_name
+  location = var.location
+  force_destroy = true
+  uniform_bucket_level_access = true
+
+  website {
+    main_page_suffix = "index.html"
+  }
+}
+
+# Add proper permissions to bucket
+resource "google_storage_bucket_iam_member" "viewer" {
+  bucket = google_storage_bucket.call_home_frontend.name
+  role   = "roles/storage.objectViewer"
+  member = "allUsers"
+}
+
+resource "google_storage_bucket_iam_member" "admin-iam" {
```

Comment:
Do use a more specific name for this, in the use_case_then_name style, like `frontend_bucket_viewer`. The reason for that is that terraform addresses are global (so the chance of conflict is high) and you don't always get to see the full details of the resource (so the name has to be self-descriptive to aid understanding).